### DISCUSSION TITLE
Board View

### DISCUSSION THREAD

**Discussion by marco-ith** [marcusolsson/obsidian-projects#661] [Help] [answered]
Hi! Is there a way to show a "status" even if there is not a note with that status yet?
Also, apparently I can't use the property type "List" as "status data" and tbh this would be amazing.

Example:
A todo status (property list) with three elements in it:
- Inbox
- In progress
- Completed

I would like to use this data type in my board view and I would like to see 3 columns (inbox, in progress, completed) even if I don't have any note with that status. Like a real kanban.
Is that possible??

I hope it was clear, if not please tell me what's not clear and I will try to write it in another way!

Thanks!

Marco

**Comment by Acylation** [comment, marked as answer]
You can config the field in table view, and set predefined values.

<img src="https://github.com/marcusolsson/obsidian-projects/assets/73122375/a72fcaa7-c379-4a86-80e5-a83c6f773eb3" width="300"/>  
<img src="https://github.com/marcusolsson/obsidian-projects/assets/73122375/ce0ebbef-f34c-49f9-8c5f-eaf8289ae18a" width="400"/>

And here's how it looks like in the board view
![image](https://github.com/marcusolsson/obsidian-projects/assets/73122375/406ce089-0c17-49b0-b7b6-e87d85d0eef8)

**Reply by Acylation** [reply]
Maybe this thread is also helpful. https://github.com/marcusolsson/obsidian-projects/issues/510

Please notice that the placeholder and filter are possibly unessary, you can test the predefined field as mentioned above first.

**Reply by marco-ith** [reply]
Amazing! The thing I was looking for. Thank you @Acylation

**Reply by oscartorres-10** [reply]
Why can't I see the Options for pre defined values?

![can't see Options in Configure field Obsidian Projects plugin](https://github.com/marcusolsson/obsidian-projects/assets/38195378/30cc3771-9f5e-4dd8-b9ef-2ce08db131fe)

**Reply by marco-ith** [reply]
Hi, are you sure the "status" property is a "text" property and not a "list" property? Also, check if there is one note (or more) with an error in the "status" property.

**Reply by oscartorres-10** [reply]
I had them as a list, that was the problem. It was confusing that in that modal (screenshot) it said Type text.

**Comment by Kamilkopec** [comment]
Exactly what I was searching for.
Thansk!

**Reply by TheRealFREDP3D** [reply]
Hi there! In Obsidian, you can achieve a Kanban-style board view even if you don't have notes with each status by using a combination of plugins and settings.

## Step-by-step guide

1. **Create a Template Note:**
   Start by creating a template note that includes your status data. In your case, a note with a list of todo statuses: Inbox, In Progress, Completed.

2. **Use the Kanban Plugin:**
   Obsidian has a community plugin called "Kanban" that you can use to create a Kanban board. Install it by going to Settings -> Community plugins, searching for "Kanban," and enabling it.

3. **Configure Kanban Board:**
   - Once the plugin is installed, go to the settings and configure it.
   - You can set up columns based on your todo statuses (Inbox, In Progress, Completed).
   - Map each column to the corresponding status from your template note.

4. **Create Placeholder Notes:**
   To have the columns display even when there are no notes with that status, you can create placeholder notes for each status. These can be empty notes or have some default content.

5. **Use Backlinks:**
   Leverage Obsidian's backlinks feature. In each placeholder note, create backlinks to your template note with the corresponding status. This will give the appearance of notes in each column.

6. **List as a Property Type:**
   Unfortunately, as of my last knowledge update in January 2022, Obsidian does not support using the property type "List" directly for status data. However, you can still use plain text and create tags or headers for your statuses.

Remember to check the Obsidian forums or documentation for any updates or new plugins that may provide additional features after my last update.

I hope this helps! If you have any more questions or if there have been changes to Obsidian's features, feel free to ask. Happy organizing!
